.circleci/config.yml

```diff
@@ -1,73 +1,110 @@
+references:
+  shared-environment: &shared-environment
+    CIRCLE_ARTIFACTS: /tmp/artifacts
+    CIRCLE_TEST_REPORTS: /tmp/test_results
+
+  docker-image: &docker-image
+    - image: circleci/node:10.10.0-browsers
+
+  setup-results-and-artifacts: &setup-results-and-artifacts
+    name: Create Directories for Results and Artifacts
+    command: |
+      mkdir -p                             \
+        "$CIRCLE_ARTIFACTS/translate"      \
+        "$CIRCLE_TEST_REPORTS/client"      \
+        "$CIRCLE_TEST_REPORTS/eslint"      \
+        "$CIRCLE_TEST_REPORTS/integration" \
+        "$CIRCLE_TEST_REPORTS/server"
+
+
+  restore-git-cache: &restore-git-cache
+    name: Restore git cache
+    key: v1-20180524-git
```

Comment:
Just moved, but it's just a unique cache key.

I _think_ we should be using something more dynamic because CI caches are never overwritten, but I haven't included that in this PR.